src/components/statusHandler/StatusHandler.tsx

```diff
@@ -2,23 +2,45 @@ import React from "react";
 import { MadieAlert } from "@madie/madie-design-system/dist/react";
 import "./StatusHandler.scss";
 
-const StatusHandler = ({ error, errorMessage, testDataId }) => {
+const StatusHandler = ({ error, errorMessages, testDataId }) => {
   {
     if (error) {
-      if (errorMessage) {
-        return (
-          <MadieAlert
-            data-testid="generic-error-text-header"
-            type="error"
-            content={
-              <h3 aria-live="polite" role="alert" data-testid={testDataId}>
-                {errorMessage}
-              </h3>
-            }
-            canClose={false}
-          />
-        );
+      if (errorMessages) {
```

Comment:
Can these 2 conditions be grouped together, as for (!errorMessages) there is no else.